cli/test/test_runner_kubernetes.py

```diff
@@ -0,0 +1,58 @@
+import os
+import unittest
+
+import utils as testutils
```

Comment:
also, I forgot to mention, this module disappeared in one of our recent changes to `master`

Reply:
Noted